tests/index.js

```diff
@@ -13,5 +15,7 @@ function init() {
   
   // kakapoSpecs()
   // dbSpec();
-  routerSpec();
+  // routerSpec();
+  // responseSpec();
+  requestSpec();
```

Comment:
We should have separate file for tests that have all tests always being run for travis :)